docs/usage/configuration/config_documentation.md

```diff
@@ -3014,6 +3014,13 @@ Options for each entry include:
      which is set to the claims returned by the UserInfo Endpoint and/or
      in the ID Token.
 
+* `backchannel_logout_enabled`: set to `true` to process OIDC Back-Channel Logout notifications. 
+  Those notifications are expected to be received on `/_synapse/client/oidc/backchannel_logout`.
+  Defaults to `false`.
+
+* `backchannel_logout_ignore_sub`: by default, the OIDC Back-Channel Logout feature checks that the
+  `sub` claim matches the subject claim received during login. This check can be disabled by setting
+  this to `true`. Defaults to `false`.
```

Comment:
Should this *never* be set if you're using `sub` as your subject claim?

```suggestion
* `backchannel_logout_ignore_sub`: by default, the OIDC Back-Channel Logout feature checks that the
  `sub` claim matches the subject claim received during login. This check can be disabled by setting
  this to `true`. Defaults to `false`.

  You might want to disable this if the `subject_claim` returned by the mapping provider is not `sub`.
```

----

Alternately, there isn't a way to configure the subject claim for backchannel logouts, correct?

Reply:
No, there isn't. The `sub` check is more of a sanity check, since `sid` are also supposed to be unique within a provider.

Also, you should be using something else than `sub` as subject claim only if you have really good reasons to do it (or if you're talking to an OAuth but non-OIDC provider like GitHub)